todo_app/data/mongo_items.py

```diff
@@ -28,3 +28,29 @@ def add_card(title):
     card = {"name": title, "state": "To Do"}
     COLLECTION.insert_one(card)
     return 0
+
+def get_myrole(ClientID):
+    #Authorised Writers
+    if ClientID.strip() in {'65459782','1'}:
+        return ""
```

Comment:
Would it make more sense to return "writer" here? This feels more natural (I'll make a comment re disabling the inputs later!)

Reply:
Done, it was a little cheat to use it in the html but yes this would be better